build.gradle

```diff
@@ -214,6 +214,7 @@ sonarqube {
     property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/sscs/HearingsApiApplication.java," +
             "src/main/java/uk/gov/hmcts/reform/sscs/config/**," +
             "src/main/java/uk/gov/hmcts/reform/sscs/model/**," +
+            "src/main/java/uk/gov/hmcts/reform/sscs/service/hearings/CreateHearingCaseUpdater.java," + // can be removed when deleting HearingService
```

Comment:
temporarily added sonar exclusion as some code is duplicated from existing HearingsService. This will be removed when cleaning up the feature toggle